webhooks/ssp_webhook.go

```diff
@@ -120,14 +121,27 @@ func (s *sspValidator) validateSspObject(ctx context.Context, ssp *sspv1beta3.SS
 }
 
 func validateCluster(ssp *sspv1beta3.SSP) error {
-	if ptr.Deref(ssp.Spec.EnableMultipleArchitectures, false) {
-		cluster := ssp.Spec.Cluster
-		if cluster == nil {
-			return fmt.Errorf(".spec.cluster needs to be non-nil, if multi-architecture is enabled")
-		}
+	if ptr.Deref(ssp.Spec.EnableMultipleArchitectures, false) && ssp.Spec.Cluster == nil {
```

Comment:
can you please add unit test which would test this condition and error returned?